### PR TITLE
Added documentation

### PR DESCRIPTION
**Pull request by ankitsin**

### CODE REVIEW

File: cities_light/downloader.py [hunk 1]
```diff
@@ -25,18 +25,20 @@ def download(self, source, destination, force=False):
...
         if not self.needs_downloading(source, destination, force):
             logger.warning(
                 'Assuming local download is up to date for %s', source)
             return False
-
+        # If the files are diffrent, download/copy happens
         logger.info('Downloading %s into %s', source, destination)
         source_stream = urlopen(source)
-        with open(destination, 'wb') as local_file:
+        with open(destination, 'wb') as local_file:     # wb: open as write and binary mode
```
Comment (jpic) [inline cities_light/downloader.py]: Could we have 2 spaces only before the hash as dictated by pep8 please ?

File: cities_light/geonames.py [hunk 2]
```diff
@@ -33,15 +34,16 @@ def __init__(self, url, force=False):
 
         destination = os.path.join(DATA_DIR, destination_file_name)
         exists = os.path.exists(destination)
-
+        #if the file is a zipped file then extract it
         if url.split('.')[-1] == 'zip' and not exists:
             self.extract(self.file_path, destination_file_name)
-
+
```
Comment (jpic) [inline cities_light/geonames.py]: Could we not add the extra spaces here please ?

File: cities_light/downloader.py [hunk 3]
```diff
@@ -58,15 +61,17 @@ def needs_downloading(self, source, destination, force):
         """Return True if source should be downloaded to destination."""
         src_file = urlopen(source)
         src_size = int(src_file.headers['content-length'])
+        #getting last modified timestamp
         src_last_modified = time.strptime(
             src_file.headers['last-modified'],
-            '%a, %d %b %Y %H:%M:%S %Z'
+            '%a, %d %b %Y %H:%M:%S %Z'              # taking time with second
```
Comment (jpic) [inline cities_light/downloader.py]: Could we have 2 spaces only before the hash as dictated by pep8 please ?

**Comment by jpic** [comment]
Thanks for your contribution, could you update your patch to pass tests please ?

Thanks !

**Comment by ankitsin** [comment]
I have made changes based on your comments. Please have look.

**Pull request merged**